lib/task-data/schemas/types-installos.json

```diff
@@ -319,6 +319,22 @@
             "type": "string",
             "description": "Extra (persistent) kernel boot parameters",
             "minLength": 1
+        },
+        "PackagesArray": {
+            "description": "specify additional packages to install",
+            "type": "array",
+            "items": {
+                "type": "string"
+            },
+            "uniqueItems": true
+        },
+        "ClearDisk": {
```

Comment:
I didn't see 'clearDisk' related codes in https://github.com/RackHD/on-taskgraph/pull/343.

Reply:
Removed 'clearDisk' from the PR. Observed some failures after rebooting the server due to 'dd' of non-installation disks. Investigating other options. Will push a separate PR for cleanup of non-OS installation disks